tuijam

```diff
@@ -628,10 +628,11 @@ class App(urwid.Pile):
             config = yaml.load(f.read())
             if config['encrypted']:
                 from scrypt import decrypt
-                try:
+                if self.config_pw is not None:
```

Comment:
This change here has nothing to do with mpris, but without it it was never loading from the config.
I'm not sure how that worked on your side.

Reply:
Yup, this is a good catch. It became a problem when I added line 578 which sets that attribute to `None` in `__init__` and forgot to update how I was checking if there was a configuration password. I don't personally use the local encryption so I didn't see the problem on my end.